src/player.cpp

```diff
@@ -8316,6 +8316,7 @@ bool player::pick_style() // Style selection menu
 
     if( selection >= STYLE_OFFSET ) {
         style_selected = selectable_styles[selection - STYLE_OFFSET];
+        add_msg_if_player( m_info, _( style_selected.obj().initiate[0] ) );
```

Comment:
i would suggest writing a getter for this std::string instead of accessing it directly; that way you can apply the translation function. (and you aren't accessing the member directly)